### PR TITLE
Reader: Updates font size and line height for titles and content.

### LINKED ISSUE #5518: Reader: Readability :)
I've recently been using the iOS reader quite a bit, and have been struggling to actually read the content. I think my main gripe is the font size and the line height seem too small (I'm also getting old, so we can just blame that too)

But, I think it might be worth an audit to see how it compares to WordPress.com. I find it much easier to read the text, and find the balance between the UI elements on the detail page to be more pleasing (web on the left):

comparison

I'd vote to match up the iOS app to the web app there.

cc @mattmiklic @aerych

### PR DESCRIPTION
**Pull request by aerych**
Fixes #5518 

To test: Launch the reader. View the post list and the post detail of a post.  Confirm the titles and content are using the correct font size and line height.

Needs review: @kurzee would you do the honors?

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift [hunk 1]
```diff
@@ -456,6 +456,12 @@ final public class ReaderDetailViewController : UIViewController
 
     private func configureRichText() {
         richTextView.delegate = self
+        let fontSize = WPStyleGuide.Detail.contentFontSize
+        let lineHeight = WPStyleGuide.Detail.contentLineHeight
+        var textOptions = richTextView.textOptions
+        textOptions["DTDefaultFontSize"] = WPStyleGuide.Cards.contentFontSize
+        textOptions["DTDefaultLineHeightMultiplier"] = lineHeight / fontSize
```
Comment (kurzee) [inline WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift]: I guess we can't access the string consts here?

**Comment by kurzee** [comment]
@aerych this looks really nice with the larger fonts! Everything looks good, just the one comment on the consts, :shipit:

File: WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift (hunk 1 above)
Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift]: Er... actually... we can.  Thanks for the nudge :)

**Comment by aerych** [comment]
Thanks @kurzee!

**Pull request merged**